spec/factories/tasks.rb

```diff
@@ -5,5 +5,11 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    # 昨日に完了したタスク
+    trait :task_done_yesterday do
```

Comment:
taskのtraitなのでtaskという単語は無くて良いと思います。
またYesterdayという情報も特段強調する必要がないと思います。
```suggestion
    trait :done do
```

Reply:
1f39514 traitの記載を修正しました。